misp_stix_converter/stix-to-misp.py

```diff
@@ -1,69 +1,94 @@
 #!/usr/bin/env python3
-
-##########################
-#     WRAPPER SCRIPT     #
-#     DEALS WITH STIX    #
-#     TO MISP            #
-##########################
-
+"""Wrapper script providing STIX to MISP conversion. """
 
 import argparse
 import logging
-import pyaml
-import time
-import sys
 import os
+import sys
+import time
+
+import pyaml
 
-from misp_stix_converter.servers import misp
 from misp_stix_converter.converters.convert import load_stix
+from misp_stix_converter.servers import misp
+
+LOGGER_FORMAT = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+CONFIG_DEFAULT_PATH = "~/.misptostix/misp.login"
+MISP_CONFIG_KEY = "MISP"
+SSL_CONFIG_KEY = "SSL"
+BLAST_FROM_PAST_PENALTY_SECS = 1.0
+args_src = sys.argv
+
+
+def get_logger(options):
+    """Instantiate customized logger."""
+    log = logging.getLogger(__name__)
+    log.setLevel(logging.DEBUG if options.verbose else logging.INFO)
+    ch = logging.StreamHandler(sys.stdout)
+    ch.setLevel(logging.DEBUG if options.verbose else logging.INFO)
+    formatter = logging.Formatter(LOGGER_FORMAT)
+    ch.setFormatter(formatter)
+    log.addHandler(ch)
+
+    return log
+
+
+def parse_args(args):
```

Comment:
argparse does not need to be passed `sys.argv`

Reply:
Cf. above, but it profits for test from receiving an explicit list ...